### ISSUE TITLE
Is 1.4.9 the current version?

### ISSUE THREAD

**Issue by asr** [sphinx-doc/sphinx#3027]
[This](http://www.sphinx-doc.org/en/stable/) url shows that the current version is 1.4.9. Is it correct? If so, I couldn't find that version on the [releases](https://github.com/sphinx-doc/sphinx/releases).

**Comment by tk0miya** [comment]
Hmm, It's strange. 1.4.9 is not released yet. It is still under developing.
current release is 1.4.8.

@shimizukawa could you check this please?
I think this comes from version definition in `sphinx/__init__.py` of stable HEAD.

**Comment by shimizukawa** [comment]
> I think this comes from version definition in sphinx/**init**.py of stable HEAD.

Yes.

`stable` means "stable branch in development". It doesn't mean "latest stable released version."

**Comment by tk0miya** [comment]
I understand. But that confuses users accessed to sphinx-doc.org.
They will see the following notation at right column:

```
Current version: 1.4.9

Get Sphinx from the Python Package Index, or install it with:
```

I think we should change the behavior of the documentation or release operations.
Could you check them please?

**Comment by shimizukawa** [comment]
Current sphinx documentation on RTD has: latest, stable, 1.4.8, 1.3.6, 1.2.3, 1.1.3, 1.0.8, 1.0, master. 
For now, stable is pointing 1.4.9(in development). After releasing 1.4.9, "1.4.8" will be disabled (permalink to 1.4.8 page will be vanished) and new 1.4.9 link will be appeared. This procedure is strongly related to branching and tagging of  sphinx code repository because RTD using them as a document version.

IMO, sphinx documentation should have versions as: develop, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0.
For now, I have no idea to achieve this on RTD. I'm thinking...

**Comment by asr** [comment]
> IMO, sphinx documentation should have versions as: develop, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0.

I would like to continue seeing the three digits in the version, that is, 1.4.8, 1.3.6, 1.2.3, 1.1.3 and 1.0.8.

**Comment by tk0miya** [comment]
As a adhoc work, I changed the setting of readthedocs. Now sphinx-doc.org refers 1.4.8 branch.

**Comment by tk0miya** [comment]
Fixed by #3679
